scripts/login.js

```diff
@@ -163,9 +163,15 @@ function retrieveUrl(location) {
   }
 }
 
-function setLandingTab(URL) {
+function setLandingTab(input) {
```

Comment:
The URL parameter name was more descriptive to me, what was your reason for changing?